tokio-util/src/time/mod.rs

```diff
@@ -8,8 +8,7 @@
 //!
 //! This type must be used from within the context of the `Runtime`.
 
-use futures_core::Future;
-use std::time::Duration;
+use core::{future::Future, time::Duration};
```

Comment:
In Tokio we usually split up imports from different modules like this:
```suggestion
use std::future::Future;
use std::time::Duration;
```
The brackets are only used for items in the same module.